tests/Http/HttpJsonResponseTest.php

```diff
@@ -4,72 +4,64 @@
 
 use JsonSerializable;
 use PHPUnit\Framework\TestCase;
+use Illuminate\Http\JsonResponse;
 use Illuminate\Contracts\Support\Jsonable;
 use Illuminate\Contracts\Support\Arrayable;
 
 class HttpJsonResponseTest extends TestCase
 {
-    public function testSetAndRetrieveJsonableData()
-    {
-        $response = new \Illuminate\Http\JsonResponse(new JsonResponseTestJsonableObject);
-        $data = $response->getData();
-        $this->assertInstanceOf('stdClass', $data);
-        $this->assertEquals('bar', $data->foo);
-    }
-
-    public function testSetAndRetrieveJsonSerializeData()
+    /**
+     * @dataProvider setAndRetrieveDataProvider
+     *
+     * @param  $data
+     */
+    public function testSetAndRetrieveData($data): void
     {
-        $response = new \Illuminate\Http\JsonResponse(new JsonResponseTestJsonSerializeObject);
-        $data = $response->getData();
-        $this->assertInstanceOf('stdClass', $data);
-        $this->assertEquals('bar', $data->foo);
-    }
+        $response = new JsonResponse($data);
 
-    public function testSetAndRetrieveArrayableData()
-    {
-        $response = new \Illuminate\Http\JsonResponse(new JsonResponseTestArrayableObject);
-        $data = $response->getData();
-        $this->assertInstanceOf('stdClass', $data);
-        $this->assertEquals('bar', $data->foo);
+        $this->assertInstanceOf(\stdClass::class, $response->getData());
```

Comment:
This change makes no much sense, \stdClass::class === 'stdClass' this will always be true for global namespaced classes, I do not see the point in adding more code